frontend/start-proxy.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Might be a pain, but could we have this written in Node? (so it's cross-platform, and this bash file can run an npm script to actually work)?

Reply:
Done. Also rewrote gen_grpc_web_protos in Node.